EIPS/eip-5601.md

```diff
@@ -0,0 +1,1663 @@
+---
+eip: 5601
+title: The Extendable Pattern
```

Comment:
Please make a more descriptive title.

Reply:
Isn't that what the description is for?

Reply:
Well yes, but the title should still be descriptive. Imagine you're someone determining whether to click on your spec in google search results. The title should be enough to understand at a high level what this spec standardizes. This does not provide enough context.